torchvision/transforms/functional.py

```diff
@@ -591,7 +591,7 @@ def _get_perspective_coeffs(
         a_matrix[2 * i + 1, :] = torch.tensor([0, 0, 0, p1[0], p1[1], 1, -p2[1] * p1[0], -p2[1] * p1[1]])
 
     b_matrix = torch.tensor(startpoints, dtype=torch.float).view(8)
-    res = torch.lstsq(b_matrix, a_matrix)[0]
+    res = torch.linalg.lstsq(a_matrix, b_matrix).solution
 
     output: List[float] = res.squeeze(1).tolist()
```

Comment:
You also need to update the following:
```suggestion
    output: List[float] = res.tolist()
```